test/functionalTest/cases/519_ngsi10_entity_types_listing/contextEntityTypeAttributes.test

```diff
@@ -111,7 +111,6 @@ Date: REGEX(.*)
           <id>ConferenceRoom</id>
         </entityId>
         <contextAttributeList>
-#SORT_START
```

Comment:
In the process of adjusting the test harness, I have drop some (maybe all?) SORT_START/SORT_END clauses. Please, do a `make ft` in your machine with this branch in order to detect problems and re-introduce them if needed.